pkg/hive/job/job.go

```diff
@@ -37,7 +38,7 @@ var Cell = cell.Module(
 // A Registry creates Groups, it maintains references to these groups for the purposes of collecting information
 // centralized like metrics.
 type Registry interface {
-	NewGroup(opts ...groupOpt) Group
+	NewGroup(scope cell.Scope, opts ...groupOpt) Group
```

Comment:
needs to be updated in the documentation: https://docs.cilium.io/en/latest/contributing/development/hive/#job-groups

Reply:
Thanks, I always forget about the docs, they have also been updated.